pallets/sponsorship/src/tests.rs

```diff
@@ -2519,3 +2519,303 @@ fn sponsor_call_is_invalid_if_sponsor_account_is_running_low() {
 		);
 	});
 }
+
+#[test]
+fn compound_transation_pass() {
+	new_test_ext().execute_with(|| {
+		let pot = 3;
+		System::set_block_number(1);
+		let pot_fee_quota = 100_000_000_000;
+		let pot_reserve_quota = 100_000_000_000;
+		let pot_details = PotDetailsOf::<Test> {
+			sponsor: 1,
+			sponsorship_type: SponsorshipType::Uniques,
+			fee_quota: LimitedBalance::with_limit(pot_fee_quota),
+			reserve_quota: LimitedBalance::with_limit(pot_reserve_quota),
+			deposit: PotDeposit::get(),
+		};
+
+		Balances::make_free_balance_be(&pot_details.sponsor, pot_reserve_quota);
+
+		assert_ok!(SponsorshipModule::create_pot(
+			RuntimeOrigin::signed(pot_details.sponsor),
+			pot,
+			pot_details.sponsorship_type,
+			pot_details.fee_quota.limit(),
+			pot_details.reserve_quota.limit()
+		));
+
+		let user_fee_quota = pot_fee_quota / 10;
+		let user_reserve_quota = pot_reserve_quota / 10;
+
+		let user = 2u64;
+
+		assert_ok!(SponsorshipModule::register_users(
+			RuntimeOrigin::signed(pot_details.sponsor),
+			pot,
+			vec![user],
+			user_fee_quota,
+			user_reserve_quota
+		));
+		let user_details = User::<Test>::get(pot, user).unwrap();
+
+		let unique_create_call = vec![
+			Box::new(RuntimeCall::Uniques(pallet_uniques::Call::create {
+				collection: 0u32,
+				admin: user,
+			})),
+			Box::new(RuntimeCall::Uniques(pallet_uniques::Call::create {
+				collection: 1u32,
+				admin: user,
+			})),
```

Comment:
Can you instead add another uniques call here which relies on the successful execution of the previous one? For example can you mint an item for that collection instead?